Source/IntegrationTests/TestFiles/LitTests/LitTest/ast/defaultParameterResolution.dfy

```diff
@@ -0,0 +1,9 @@
+// RUN: ! %resolve %s --type-system-refresh > %t
+// RUN: ! %resolve %s >> %t
+// RUN: %diff "%s.expect" "%t"
```

Comment:
Instead of these 3 lines, use

    // RUN: %testDafnyForEachResolver "%s"

In addition to an `.expect` file, use a `.refresh.expect` file, if needed. Also, use `--expect-exit-code` and/or `--refresh-exit-code`.

Reply:
I don't think that's possible because the behavior is different for the two resolvers.

Reply:
Done